coral/src/app/features/topics/details/history/TopicHistory.tsx

```diff
@@ -0,0 +1,83 @@
+import { DataTable, DataTableColumn, EmptyState } from "@aivenio/aquarium";
+import { useTopicDetails } from "src/app/features/topics/details/TopicDetails";
+import { TopicOverview } from "src/domain/topic";
+
+interface TopicHistoryRow {
+  id: number;
+  logs: string; //Remarks
```

Comment:
The "logs" column is the "remarks" property? :o I am thinking we need toreview both the API definition and the copy to make it a little bit more explicit. As far as I see, this is actually the "operation type", not a log or a remark :o

Reply:
>The "logs" column is the "remarks" property? 
Yes. The column is called "Remarks" in the Angular app, but "logs" here.

Reply:
Yes, I see remarks field is used to set the operation type. We can rename the field to RequestOperationType

Reply:
> Yes, I see remarks field is used to set the operation type. We can rename the field to RequestOperationType

Keeping the column as is for now, and updating the content with a prefix of entity type